components/raftstore/src/store/fsm/store.rs

```diff
@@ -1691,30 +1696,45 @@ impl<EK: KvEngine, ER: RaftEngine> RaftBatchSystem<EK, ER> {
             None
         };
         let bgworker_remote = background_worker.remote();
+        let snap_gen_worker = WorkerBuilder::new("snap-gen-worker")
```

Comment:
Please add it to grafana board

Reply:
Updated the existing "Snap generator CPU" panel to track the snap gen worker CPU.